# AUTHOR: Neset Unver Akmandor (NUA)
# E-MAIL: akmandor.n@northeastern.edu

world_frame_name: "world"
gz_model_msg_name: "model_states"

### Options: NUA TODO: TEST AGAIN!
## "conveyor_belt"
## "normal_pkg"
## "long_pkg"
## "longwide_pkg"
## "red_cube"
## "green_cube"
## "blue_cube"
## "actor_0"
## "actor_1"
## "bin_4_dropping_task"
#name_pkgs_ign: ["conveyor_belt","red_cube","actor_0","actor_1"] 
name_pkgs_ign: ["conveyor_belt","red_cube"] 
name_pkgs_man: [] 

### Options: NUA TODO: TEST AGAIN!
## "models/point_cloud/conveyor_belt.json",
## "models/point_cloud/normal_pkg.json",
## "models/point_cloud/long_pkg.json",
## "models/point_cloud/longwide_pkg.json",
## "models/point_cloud/red_cube.json",
## "models/point_cloud/green_cube.json",
## "models/point_cloud/blue_cube.json",
## "models/point_cloud/actor_0.json",
## "models/point_cloud/actor_1.json",
## "models/point_cloud/bin_4_dropping_task.json"
scan_data_path_pkgs_ign: ["models/point_cloud/conveyor_belt.json",
                          "models/point_cloud/red_cube.json",
                          "models/point_cloud/actor_0.json",
                          "models/point_cloud/actor_1.json"]

scan_data_path_pkgs_man: []

map_resolution: 0.05

egrid_frame_name: "base_link"
egrid_resolution: 0.1

egrid_bbx_min_x: -3
egrid_bbx_min_y: -3
egrid_bbx_min_z: 0

egrid_bbx_max_x: 3
egrid_bbx_max_y: 3
egrid_bbx_max_z: 2

egrid_occ_threshold: 1
